easy_infra.yml

```
@@ -83,7 +83,7 @@ packages:
       env_vars:
       - TF_DATA_DIR
     security: *id003
-    version: 1.5.4
+    version: 1.5.5
```

Pin tf version until we iron out the terraform relicensing
```suggestion
    version: 1.5.5
    allow_update: False
```